src/pages/tools.astro

```diff
@@ -0,0 +1,59 @@
+---
+import Layout from "../layouts/Layout.astro"
+import Card from "../components/Card.astro"
+import type { ToolContent } from "../lib/content"
+import luminiteImage from "../assets/content/luminite.png"
+import replayRenamerImage from "../assets/content/replay-renamer.png"
+
+const stormgateWorldToolsContent: ToolContent[] = [
+  {
+    title: "Streamer Overlay",
+    source: "tool",
+    description: "A tool for streamers to display information about ongoing games in their broadcasts.",
+    url: "https://github.com/stormgateworld/overlay",
+    image_url: "https://raw.githubusercontent.com/stormgateworld/overlay/main/Preview.png",
+  },
+]
+
+const communityToolsContent: ToolContent[] = [
+  {
+    title: "Luminite.gg",
+    source: "tool",
+    description: "Discover and share build orders. View unit information and ladder stats.",
+    url: "https://luminite.gg/",
+    image_url: luminiteImage.src,
+    metadata: { tags: ["Build orders", "Stats", "External"] },
+  },
+  {
+    title: "Replay Renamer",
+    source: "tool",
+    description: "Python script to rename replays.",
+    url: "https://gist.github.com/acarapetis/b5e9cf199ea2a468ace2f756ac287268",
+    image_url: replayRenamerImage.src,
+    metadata: { tags: ["Replays", "External"] },
+  },
+]
+---
+
+<Layout title="Tools">
+  <div class="w-full bg-gray-800/50 backdrop-blur-lg border-b border-gray-700/50">
+    <div class="mx-auto max-w-screen-md px-4 md:px-0 py-8 gap-4 flex items-center flex-wrap justify-between">
```

Comment:
I'm really happy with everything, except these headers.

I think you can best copy over the header from the Stats page and just replace the text.

If you want to split the tools into subheading you can copy the code that says 'Factions' as a header above the two groups of tools.

Alternatively you could just show 'offical' or 'by SGW' or something as a tag maybe. I'm fine either way